package.json

```diff
@@ -197,6 +197,9 @@
       "@angular/animations": {
         "dependencies": {
           "@nginfra/angular-linking": "1.0.6"
+        },
+        "peerDependencies": {
```

Comment:
Is it possible to add a comment for a todo here? I know comments are hard to include in package.json so it might not be feasible in this case.

Reply:
Unfortunately not. The commit message is the best comment here.